docs/solr.md

```diff
@@ -0,0 +1,50 @@
+# `solr` Metrics Receiver
+
+The `solr` metrics receiver can fetch stats from a Solr server's Java Virtual Machine (JVM) via [JMX](https://www.oracle.com/java/technologies/javase/javamanagement.html).
+
+## Prerequisites
+
+In order to expose a JMX endpoint, you must set the `com.sun.management.jmxremote.port` system property. It is recommended to also set the `com.sun.management.jmxremote.rmi.port` system property to the same port. To expose JMX endpoint remotely, you must also set the `java.rmi.server.hostname` system property. By default, these properties are set in a Solr deployment's solr-env.sh file and the default Solr installation requires no JMX authentication with JMX exposed locally on 127.0.0.1:18983.
+
+## Configuration
+
+| Field                 | Default            | Description |
+| ---                   | ---                | ---         |
+| `type`                | required           | Must be `solr`. |
+| `endpoint`            | `localhost:18983`  | The [JMX Service URL](https://docs.oracle.com/javase/8/docs/api/javax/management/remote/JMXServiceURL.html) or host and port used to construct the Service URL. Must be in the form of `host:port`. Values in `host:port` form will be used to create a Service URL of `service:jmx:rmi:///jndi/rmi://<host>:<port>/jmxrmi`. |
+| `username`            | not set by default | The configured username if JMX is configured to require authentication. |
+| `password`            | not set by default | The configured password if JMX is configured to require authentication. |
+| `collection_interval` | `60s`              | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
+
+Example Configuration:
+
+```yaml
```

Comment:
It's preferred for the receiver and pipeline IDs to stay the same as the receiver type in the sample configuration. E.g.  "s/{APP}_metrics/{APP}", "s/{APP}_pipeline/{APP}"

Reply:
Thanks!